portfolio/src/main/webapp/index.html

```diff
@@ -1,34 +1,61 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <head>
-    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=0.55">
+    <meta
+      charset="UTF-8"
+      name="viewport"
+      content="width=device-width, initial-scale=0.55"
+    />
     <title>Zoe's Portfolio</title>
-    <link rel="stylesheet"
-         href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
-         integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" 
-         crossorigin="anonymous">
-    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
-    
     <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
       <div id="content">
-        <button class="navbar-toggler" 
-                type="button" 
-                data-toggle="collapse" 
-                data-target="#navbarNavAltMarkup" 
-                aria-controls="navbarNavAltMarkup" 
-                aria-expanded="false" 
-                aria-label="Toggle navigation">
-            <span class="navbar-toggler-icon"></span>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarNavAltMarkup"
+          aria-controls="navbarNavAltMarkup"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
           <div class="navbar-nav">
-            <a class="nav-item nav-link nav-link-spacing active" href="#" id="nav-link">Zoe Barth</a>
-            <a class="nav-item nav-link nav-link-spacing" href="#about-me" id="nav-link">About Me</a>
-            <a class="nav-item nav-link nav-link-spacing" href="#projects" id="nav-link">Projects</a>
-            <a class="nav-item nav-link nav-link-spacing" href="#contact-me" id="nav-link">Contact Me</a>
+            <a
+              class="nav-item nav-link nav-link-spacing active"
+              href="#"
+              id="nav-link"
+              >Zoe Barth</a
+            >
```

Comment:
Wow, prettier does not look pretty here! Looks like there's a reason though, so this is fine: https://prettier.io/blog/2018/11/07/1.15.0.html